app/scripts/modules/core/src/utils/consoleDebug.ts

```diff
@@ -18,6 +28,9 @@ export class ConsoleDebugWindow {
 }
 
 export const DebugWindow = new ConsoleDebugWindow();
+if (window) {
+  window.spinnaker = DebugWindow;
+}
```

Comment:
what's the context behind this additional assignment vs. the one on L35? Is it mostly just to make use of the new interface augmentation on `Window`? If yes, why keep the other one?

Reply:
ack, that's a bug!